lightningd/peer_control.c

```diff
@@ -542,21 +542,6 @@ static enum watch_result funding_lockin_cb(struct lightningd *ld,
 	if (!channel_tell_funding_locked(ld, channel, txid, depth))
 		return KEEP_WATCHING;
 
-	/* BOLT #7:
-	 *
-	 * A node:
-	 *   - if the `open_channel` message has the `announce_channel` bit set
-	 *     AND a `shutdown` message has not been sent:
-	 *     - MUST send the `announcement_signatures` message.
-	 *       - MUST NOT send `announcement_signatures` messages until
-	 *         `funding_locked` has been sent AND the funding transaction has
-	 *         at least six confirmations.
-	 *   - otherwise:
-	 *     - MUST NOT send the `announcement_signatures` message.
-	 */
-	if (!(channel->channel_flags & CHANNEL_FLAGS_ANNOUNCE_CHANNEL))
-		return DELETE_WATCH;
-
```

Comment:
This change is probably OK, but I'd like something in the commit msg justifying its deletion; it seems orthogonal?

Reply:
reverted. I was under the mistaken assumption that this would prevent private channels from being locked. Turns out that check is the line above...